services/api/src/models/Users.ts

```diff
@@ -42,18 +42,24 @@ export function subscribeToCreatorsList(mc: any, email: string) {
   });
 }
 
+interface IUserQuestType {
+  details: Quest;
+  lastPlayed: Date;
+}
+
 export interface UserQuestsType {
-  [questId: string]: {
-    details: Quest;
-    lastPlayed: Date;
-  };
+  [questId: string]: IUserQuestType;
 }
 
-export function getUserQuests(db: Database, id: string): Bluebird<UserQuestsType> {
+export function getUserQuests(db: Database, id: string, questIds?: string[]): Bluebird<UserQuestsType> {
```

Comment:
Added this questId to filter the user posts on the bases of quests on which user had given the feedback.

Reply:
Would be good to add this as a comment to the code so that we remember why this happened in the future :)

Reply:
Done